medical/__openerp__.py

```diff
@@ -37,7 +42,7 @@
 ##############################################################################
 {
     'name': 'Odoo Medical',
-    'version': '8.0.1.1.0',
+    'version': '9.0.2.0.0',
```

Comment:
@pedrobaeza - Just so I'm 100% - the upgrade from Odoo is already considered breaking & in requirement of migrations, hence the downgrade to 1.0.0 internal versioning?